Formula/homeshick.rb

```diff
@@ -0,0 +1,122 @@
+class Homeshick < Formula
+  desc "Git dotfiles synchronizer written in bash"
+  homepage "https://github.com/andsens/homeshick"
+  url "https://github.com/andsens/homeshick/archive/1.0.0.tar.gz"
+  sha256 "8bd3c46f1cfd68d82d97fa72a68a07c966092c77f276d1335cb390b2ec6062bf"
+  head "https://github.com/andsens/homeshick.git"
+
+  option "with-fish", "Build fish bindings"
+  option "with-csh", "Build csh bindings"
+  option "without-bash", "Build without bash bindings"
+
+  if (build.without? "bash") && (build.without? "fish") && (build.without? "csh")
```

Comment:
This should be inside `def install`, but either way I'd probably rather we just build at least one of these options. It seems reasonable to assume people would want `bash` support?